### DISCUSSION TITLE
GitFlow & Rules

### DISCUSSION THREAD

**Discussion by ZompaSenior** [ConoscereLinux/asso-django#5] [General]
Can we work on a Gitflow way (main, develop, feature)?
Do you think it's better that everyone forks the project, and works with pull requests?
@giobber did you have any ideas on these topics?

**Comment by giobber** [comment]
It depends...

Gitflow if used wrongly is a living hell, but I've used it only a couple of time

At the moment I'm more concerned in apply the KISS approach and do the minimum necessary to get results

How do you have used gitflow in the past?

**Reply by ZompaSenior** [reply]
I use it everyday at work.
At the moment, maybe, it is not necessary to implement too much branch, but maybe we can work on forked repo with pull request, after having prepared some consistent implementation.
Committing code in a wild way, maybe it is not a good way to work together, do you agree?

Maybe we can simply create a branch named develop, and merge the pull request on it, do you like it?

In a second step (when there is something working), we can make some feature/something, for the new functionality. Normally nobody had to work on the develop branch, nor on the main/release one.
